pkgs/by-name/la/lazyjj/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  binutils-unwrapped,
+  fetchFromGitHub,
+  makeWrapper,
+  jujutsu,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "lazyjj";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "Cretezy";
+    repo = "lazyjj";
+    rev = "v${version}";
+    hash = "sha256-VlGmOdF/XsrZ/9vQ14UuK96LIK8NIkPZk4G4mbS8brg=";
+  };
+
+  cargoHash = "sha256-TAq9FufGsNVsmqCE41REltYRSSLihWJwTMoj0bTxdFc=";
+
```

suggestion: Add a minimal `passthru` test with a version check:

```suggestion

  passthru.tests.version = testers.testVersion {
    package = lazyjj;
  };
  
```

(requires `testers` and `lazyjj` to be added to the args list).